provision/minikube/Taskfile.yaml

```diff
@@ -68,6 +68,7 @@ tasks:
     sources:
```

Comment:
to test this, would it just be enough to run `task` or `task -f` on an existing cluster ? I am making sure, if a total rebuild of the minikube VM is necessary or not

Reply:
To test this, running `task` is sufficient. Sorry for now putting this in the PR in the first place.